{
  "comments": [
    {
      "key": {
        "uuid": "e7e1b169_b339a6a7",
        "filename": "dump/dump_collect.cpp",
        "patchSetId": 2
      },
      "lineNbr": 100,
      "author": {
        "id": 1000089
      },
      "writtenOn": "2021-10-25T08:13:55Z",
      "side": 1,
      "message": "we are already creating PEL do we still need to rethrow the error",
      "range": {
        "startLine": 94,
        "startChar": 0,
        "endLine": 100,
        "endChar": 9
      },
      "revId": "8b5302cf1325317c32aab4a88ba08f7619f506cf",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}